server/db/models/orderItem.js

```diff
@@ -0,0 +1,17 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const OrderItem = db.define('orderItem', {
+  serial: {
+    type: Sequelize.STRING
```

Comment:
Serial: Unique and required for the item?

Reply:
I was going based on your slack convo last night re. serial numbers.  We can take it out if not needed but I think it's good to have.  Not required now but we can.